pkg/deploy/deploy_test.go

```diff
@@ -748,38 +747,7 @@ func TestDeployConfigsWithDeploymentErrors(t *testing.T) {
 		deploy.EnvironmentInfo{Name: env}: deploy.DummyClientSet,
 	}
 
-	t.Run("[non-parallel] deployment error - stop on error", func(t *testing.T) {
-		t.Setenv(featureflags.DependencyGraphBasedDeployParallel().EnvName(), "false") // parallel deploy always continues
-
-		err := deploy.DeployConfigGraph(p, c, deploy.DeployConfigsOptions{})
-		assert.Error(t, err)
-
-		envErrs := make(errors.EnvironmentDeploymentErrors)
-		assert.ErrorAs(t, err, &envErrs)
-		assert.Len(t, envErrs, 1)
-		assert.Len(t, envErrs[env], 1)
-		var depErr errors.DeploymentErrors
-		assert.ErrorAs(t, envErrs[env][0], &depErr)
-		assert.Equal(t, 1, depErr.ErrorCount, "Expected deployment to return after the first error")
-	})
-
-	t.Run("[non-parallel] deployment error - continue on error", func(t *testing.T) {
-		t.Setenv(featureflags.DependencyGraphBasedDeployParallel().EnvName(), "false")
-
-		err := deploy.DeployConfigGraph(p, c, deploy.DeployConfigsOptions{ContinueOnErr: true})
-		assert.Error(t, err)
-
-		envErrs := make(errors.EnvironmentDeploymentErrors)
-		assert.ErrorAs(t, err, &envErrs)
-		assert.Len(t, envErrs, 1)
-		assert.Len(t, envErrs[env], 1)
-		var depErr errors.DeploymentErrors
-		assert.ErrorAs(t, envErrs[env][0], &depErr)
-		assert.Equal(t, 2, depErr.ErrorCount, "Expected deployment to continue after the first error and count errors for both invalid configs")
-	})
-
 	t.Run("[parallel] deployment error - always continues on error", func(t *testing.T) {
```

Comment:
nit:
```suggestion
	t.Run("deployment error - always continues on error", func(t *testing.T) {
```

Reply:
thx, this means we can actually also delete the `ContinueOnError` Options field

Reply:
I think it's still used for environments? We deploy to envs in sequence don't we? And use cont-on-error there?